src/customTheme.ts

```diff
@@ -1,6 +1,71 @@
-import { extendTheme } from "@chakra-ui/react";
-import { Button } from "./components/Button";
+import { extendTheme} from "@chakra-ui/react";
+import { Button} from "./components/Button";
 
+const baseText = {
+	baseStyle:{
+		textAlign: "center",
+		fontSize: "30",
+		color: "white",
+		lineHeight: "1.4",
+		
+	},
+};
+export const logoStyle = {
+	position: "absolute" as const,
+	top: "5px",
+	left: "5px",
+	
+};
+export const pictureBox = {
+	color: "pink",
+	top: "50px",
+	background: "gray",
+	width: "300px", 
+    height: "400px",
+	marginTop: "200px"
+}; 
+export const poloroidBox = 
+{
+	width: "200px",
+	height: "200px",
+	marginTop: "100px", 
+	background: "gray",
+}; 
+export const sponsorBox = 
+{
+	width: "550px",
+	height: "100px",
+	background: "lightblue", 
+	marginLeft: "20px", 
+	marginRight: "20px"
+	
+};
+export const bigBox = 
```

Comment:
small nit that honestly might not matter if we're getting rid of these boxes later, but try to make these style names more descriptive! like how you did for sponsorBox, polaroidBox, etc.